frontend/src/components/AddPost/AddPost.cy.js

```diff
@@ -0,0 +1,23 @@
+import AddPost from './AddPost';
+const navigate = () => {};
+
+describe('AddPost', () => {
+  it('returns a ', () => {
+    cy.mount(<AddPost navigate={navigate} />);
+
+    cy.intercept('POST', '/posts', {
```

Comment:
Bravo for getting stuck into Cypress! Im curious as to what exactly this is testing, would be great to hear about it from your perspective,